package/tests/test_models/test_graph/test_graph_nodes.py

```diff
@@ -551,6 +553,15 @@ def test_load_metrics_versioned_data(self, metrics_filepath):
             == mock_metrics_json
         )
 
+    def test_load_latest_metrics(self, metrics_filepath):
+        dataset = MetricsDataSet(filepath=f"{metrics_filepath}")
+        mock_metrics_json = {
+            "recommendations": 0.200383330721228,
+            "recommended_controls": 0.250479163401535,
+            "projected_optimization": 0.30057499608184196,
+        }
+        assert DataNodeMetadata.load_latest_metrics_data(dataset) == mock_metrics_json
```

Comment:
would it be possible to write a test to check that if we write another version of this dataset and load it again, it picks up the change?